packages/pyright-internal/typeshed-fallback/stubs/uWSGI/METADATA.toml

```diff
@@ -9,7 +9,4 @@ extra_description = """\
 """
 
 [tool.stubtest]
-# Run stubtest on MacOS as well, to check that the
-# uWSGI-specific parts of stubtest_third_party.py
-# also work there
-platforms = ["linux", "darwin"]
```

Comment:
hmm. not sure why this is happening. pylance shouldn't be changing this

Reply:
Yeah we should back these parts out. Or just manually do it.

Reply:
done